src/woo_search/search_index/api/serializers/publications.py

```diff
@@ -143,6 +143,10 @@ def validate(self, attrs: DocumentIndexType):
         if not identifiers and identifier:
             attrs["identifiers"].append(identifier)
 
+        # TODO: remove this when `gepubliceerd_op` is required
+        if "gepubliceerd_op" not in attrs:
+            attrs["gepubliceerd_op"] = attrs["registratiedatum"]
```

Comment:
```suggestion
        if not in attrs.get("gepubliceerd_op"):
            attrs["gepubliceerd_op"] = attrs["registratiedatum"]
```
covers the case where it *is* sent, but set to `null`